requirements.txt

```diff
@@ -1,8 +1,13 @@
-    pytket == 1.4.1
+[comment]: # if version are updated, update the version in the build-manual file
+[comment]: # and the check-examples.yml, too.
+[comment]: # this duplication is currently necessary because of an issue with the pyjwt
+[comment]: # version required from qcs-api-client < 0.21.0
+
+    pytket == 1.4.3
     pytket-cirq == 0.25.0
-    pytket-projectq == 0.23.0
     pytket-pyquil == 0.25.0
+    pytket-projectq == 0.23.0
```

Comment:
Is this change of order necessary? (They were alphabetical.)

Reply:
No it is not, have ordered them again alphabetical